packages/sanity-plugin/src/sanityDocuments/collection.tsx

```diff
@@ -94,7 +94,7 @@ export const createCollectionDocument = ({
         const itemTitle = sourceData ? title || sourceData.title : title
         const alt = `Image for ${itemTitle}`
         const src = sourceData?.image?.w100
-        const subtitle = archived ? '📁 Archived' : undefined
+        const subtitle = archived ? '📁 Archived' : sourceData?.shopName || ''
```

Comment:
Could we only include the shopName in the subtitle if the user has multiple storefronts added?

Reply:
If product is archived it probably do not have a "store" anymore (deleted, archived or so)
In my opinion is better to keep the "Archived" label, what do you think?

Reply:
Ah ok get it, you don't want to include if there is only "one" store connected.
Is not easy because we should make a count query to the secrets document... any better ideas?